package.json

```
@@ -65,6 +65,7 @@
   },
   "dependencies": {
     "@cloudinary/url-gen": "^1.10.2",
+    "cloudinary-video-analytics": "1.1.0",
```

We should aim the keep semantic versioning there, and so it is a good idea to get minor versions here. 
```suggestion
    "cloudinary-video-analytics": "^1.1.0",
```

---

👍